imports/plugins/core/ui/client/containers/inventoryBadge.js

```diff
@@ -30,7 +28,7 @@ const composer = (props, onData) => {
       label = "Sold Out!";
       i18nKeyLabel = "productDetail.soldOut";
     } else if (inventoryManagement) {
-      if (lowInventoryWarningThreshold >= inventoryQuantity) {
+      if (lowInventoryWarningThreshold >= variant.inventoryAvailableToSell) {
```

Comment:
The server badge code does `<=`, so this should be `>` right? Or reverse these and do `<=` to match server